packages/core/src/common/commands-common.ts

```diff
@@ -9,6 +9,16 @@ import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
 
+const enjson = require('./i18n/en.json');
+const esjson = require('./i18n/es.json');
+const frjson = require('./i18n/fr.json');
+const Globalize = require("globalize");
+
+// Load potential languages
+Globalize.loadMessages(enjson);
+Globalize.loadMessages(esjson);
+Globalize.loadMessages(frjson);
```

Comment:
We can't have dynamic requires however like

function require_lang(lang: string) {
  var  lang = require(lang);
}

Since webpack won't be able to bundle that..

We could however have function for the Globalize.loadMessages part ...

Or did you know of a way for a dynamic require with webpack @svenefftinge ?